Update new keystore migration note (#4979)
wso2/docs-is: en/includes/deploy/security/keystores/configure-keystores.md

### Configure the internal keystore

!!! warning

>     Adding a new keystore for internal data encryption for an existing deployment will make already encrypted data unusable. If asymmetric key encryption or symmetric key encryption with a new secret is used, an appropriate data migration effort is needed.

Reviewer: ```suggestion
    If asymmetric key encryption is used, adding a new keystore for internal data encryption for an existing deployment will make already encrypted data unusable. In such cases, an appropriate data migration effort is needed.
```
